src/cephadm/tox.ini

```diff
@@ -41,7 +41,7 @@ commands=pytest {posargs}
 
 [testenv:mypy]
 basepython = python3
-deps = mypy==0.790
+deps = mypy==0.812
```

Comment:
i assume that we stick with 0.812 instead of 0.901 for good reasons.

Reply:
no. overlooked it